spring-rabbit/src/test/java/org/springframework/amqp/rabbit/connection/CachingConnectionFactoryIntegrationTests.java

```diff
@@ -92,6 +94,9 @@ public void open() {
 
 	@After
 	public void close() {
+		if (!this.connectionFactory.getVirtualHost().equals("non-existent")) {
```

Comment:
Doh... Need to read the code locally before asking such a stupid questions...
Thanks.
So, merging with that simple `@since` fix